### DISCUSSION TITLE
[FEATURE] 加入非root之docker实体的支持

### DISCUSSION THREAD

**Discussion by ckykenken** [1Panel-dev/1Panel#5549] [Ideas]
### 1Panel 版本

1.10.10

### 请描述您的需求或者改进建议

我的docker实体是透过此指令安装的：
```curl -fsSL https://get.docker.com/rootless | sh```
安装完成后才发现它只辨识有root之安装的容器 (即只辨认 /var/lib/docker 的资料)，而透过root安装的docker有安全隐患

**Comment by wanghe-fit2cloud** [comment]
感谢反馈。
1Panel 的其他功能（如工具箱和主机-文件管理）也需要 root 权限，因此目前我们无法适配非 root 用户的各种细节。
> 我们可能会在 v2.0 版本重构时考虑这些需求。
